src/textures/Texture.js

```diff
@@ -98,6 +98,12 @@ class Texture extends EventDispatcher {
 
 	}
 
+	equals( other ) {
```

Comment:
Can we avoid having an `equals()` method in `Texture`?

I would expect an implementation that compares all properties, not just version and uuid.

Reply:
Ok